Description: Use global object this
 This is required to match the original upstream build configuration
 and makes it work on both node and web targets
--- a/Gruntfile.js
+++ b/Gruntfile.js
@@ -77,12 +77,14 @@
       handlebars: {
         entry: './dist/cjs/handlebars.js',
         output: {
+          globalObject: 'this',
           filename: 'handlebars.js'
         }
       },
       runtime: {
         entry: './dist/cjs/handlebars.runtime.js',
         output: {
+          globalObject: 'this',
           filename: 'handlebars.runtime.js'
         }
       }
